org.eclipse.xtend.ide.tests/batch-compiler-data/ant/test.ant

```diff
@@ -35,8 +35,12 @@
 			<include name="org.eclipse.emf.common*.jar" />
 			<include name="org.eclipse.emf.codegen*.jar" />
 			<include name="com.google.guava*.jar" />
+			<include name="*guava*32*.jar" />
 			<include name="com.google.inject*.jar" />
+			<include name="guice*.jar" />
+			<include name="*failureaccess*jar" />
```

Comment:
I wonder why we need to list failureaccess but not the other Guava deps. Not actionable though.

Reply:
cause this is a runtime dep that "explodes"